doc/input-output-reference/src/overview/group-zone-equipment.tex

```diff
@@ -83,7 +83,7 @@ \subsection{ZoneHVAC:AirDistributionUnit}\label{zonehvacairdistributionunit}
 
 \end{itemize}
 
-Connections between the air distribution unit, the supply air duct, and the zone are specified in the input syntax for the air distribution unit and the \hyperref[airloophvaczonesplitter]{AirLoopHVAC:ZoneSplitter}. The input syntax also explicitly defines an outlet identifier. This implies a connection to a zone through a \hyperref[nodelist]{NodeList} for zone inlets (see the \hyperref[zonehvacequipmentconnections]{ZoneHVAC:EquipmentConnections} statement). The air distribution unit is limited to one combined component-controller unit; because controls are normally based on the zone thermostat and can work in parallel or series in complex fashion. Since the control and the flow resolution can be complex, each air distribution unit is unique in addressing these combinations and therefore only one is allowed per zone.
+Connections between the air distribution unit, the supply air duct, and the zone are specified in the input syntax for the air distribution unit and the \hyperref[airloophvaczonesplitter]{AirLoopHVAC:ZoneSplitter}. The input syntax also explicitly defines an outlet identifier. This implies a connection to a zone through a \hyperref[nodelist]{NodeList} for zone inlets (see the \hyperref[zonehvacequipmentconnections]{ZoneHVAC:EquipmentConnections} statement). Each air distribution unit is essentially a combined component-controller. Since controls are normally based on the zone thermostat, they can work in parallel or series in complex fashion. So, while it may be complex, it is possible in EnergyPlus to have more than one air distribution unit per zone.
```

Comment:
An alternative last sentence:
Each air distribution unit operates to meet all or part of the remaining thermostat load as specified in ZoneHVAC:EquipmentList.

Reply:
Sounds much better!  Replaced lat sentence with your suggestion.  Shows up in next commit.